### ISSUE TITLE
account_transactions & account_transactions_stats , when will those methods be reliable again ?

### ISSUE THREAD

**Issue by FrRichard** [ripple/ripple-data-api#22]
I didn't notice any weird result with account_transaction. (account_transaction_stat is not right ok).

Anyway those methods are extremely useful for many things when do you they'll be reliable again ? It would be nice to have a little bit of visibility on this api. I spent a lot of hours working on it and then all this time may be lost. 

My question is very simple, will those methods be fixed ? If yes I'll keep working on it with the current format and wait for reliable result. PLease give us an approximation for the release. (1 week ? 1 month ? 1 year ?). Thanks a lot.

**Comment by shekenahglory** [comment]
these endpoints wont be fixed - there will be an equivalent on the historical API at history.ripple.com with a slightly different format.  We don't have an equivalent planned right now for account_transaction_stats.  what is your use case for that endpoint?

**Comment by FrRichard** [comment]
Yes after posting I figured out I could have the same result using the historical api ( some parsing has to been done but nothing terrible).
account_transaction_stats was just a nice metric to have an overview of an account activity. I could use historical api (type=offer_create ..offer_cancel...).

Do you think it would be possible to have more result per call (with the historical api) , it is limited to 1000 and it needs a lot of call to analyse long period for very active accounts. 

 Thanks for your quick answer.
